### PR TITLE
Library creation refactoring

### PR DESCRIPTION
**Pull request by florian-huber**
- avoid unexpected behavior --> user must give filename for library output files
- replace `app_helpers` functions (gave errors when running from Jupyter notebook)
- remove fixed embedding vector size
- removed temporary file writing for embeddings to speed up this step (todo: check if really much faster!)
- some minor refactoring

### CODE REVIEW

File: ms2query/create_library_files.py [hunk 1]
```diff
@@ -10,50 +11,48 @@
 from ms2query.create_sqlite_database import make_sqlfile_wrapper
 from ms2query.spectrum_processing import minimal_processing_multiple_spectra, \
     create_spectrum_documents
-from ms2query.app_helpers import load_pickled_file
 
 
 class CreateFilesForLibrary:
     def __init__(self,
                  pickled_spectra_file_name: str,
+                 output_file_sqlite: str,
```
Comment (florian-huber) [inline ms2query/create_library_files.py:19]: We could even decide to call this `output_base_filename` and remove the options to name files individually.

File: ms2query/create_library_files.py [hunk 2]
```diff
@@ -1,59 +1,58 @@
...
 from ms2deepscore.models import load_model as load_ms2ds_model
 from spec2vec.vector_operations import calc_vector
+from ms2query.utils import load_pickled_file
 from ms2query.create_sqlite_database import make_sqlfile_wrapper
 from ms2query.spectrum_processing import minimal_processing_multiple_spectra, \
     create_spectrum_documents
-from ms2query.app_helpers import load_pickled_file
 
 
 class CreateFilesForLibrary:
```
Comment (florian-huber) [inline ms2query/create_library_files.py:16]: The name made me more think of a function (which would also have been a viable solution to create the library files). If we use a class for it, maybe we can name it something like `LibraryFilesCreator`? "CreateFilesForLibrary" doesn't make clear when those files are created. Here the user actually has to call additional methods first before something is actually created.

Comment (florian-huber) [reply ms2query/create_library_files.py]: I changed the name to `LibraryFilesCreator`, feel free to pick/suggest another name!

File: ms2query/create_library_files.py (hunk 1 above)
Comment (niekdejonge) [reply ms2query/create_library_files.py]: I changed this and added that to this pull request

**Pull request merged**